code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -38,9 +35,13 @@
 			icon_state = "[caste] Running"
 		else
 			icon_state = "[caste] Walking"
-
 	update_fire() //the fire overlay depends on the xeno's stance, so we must update it.
 
+/mob/living/carbon/Xenomorph/proc/update_action_button_icons()
```

Comment:
This seems like something worth having at the same place you can have actions instead of being a xenomorph specific proc

Reply:
Made an `item`-specific one, that was the only other instance using it.

Reply:
It doesn't matter what else is currently using it, it should be defined at whatever level can have action buttons